package.json

```diff
@@ -44,11 +44,14 @@
     "http-status-codes": "^2.2.0",
     "launchdarkly-js-client-sdk": "^2.22.1",
     "lodash": "^4.17.21",
+    "pdfjs-dist": "3.4.120",
     "pinia": "^2.0.33",
     "pinia-class": "^0.0.3",
     "regenerator-runtime": "^0.13.9",
     "sbc-common-components": "3.0.5",
+    "tiptap-vuetify": "^2.24.0",
     "uuid": "^9.0.0",
+    "v-sanitize": "^0.0.11",
```

Comment:
Is this compatible with Vue3?

Reply:
Probably not, but it's fairly easy to swap out, it just removes HTML tags from an input

Reply:
Please add another ticket to the Vue3 epic:
https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/13576

Reply:
New ticket is bcgov/entity#16490.